src/test/resources/Homework23.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework23">
+    <parameter name="BaseURL" value="https://qa.koel.app"/>
+    <test name="Homework23" preserve-order="false">
+        <classes>
+            <class name="Homework23">
+                <include name = "playSong"/>
+            </class>
+        </classes>
+    </test>
+</suite>
```

Comment:
You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/e7e1cbd2-989d-4adb-a693-2ae9e637dace)